app/manifest.json

```diff
@@ -59,14 +59,19 @@
     "unlimitedStorage",
     "clipboardWrite",
     "http://localhost:8545/",
-    "https://*.infura.io/"
+    "https://*.infura.io/",
+    "activeTab",
+    "webRequest",
+    "*://*.eth/"
   ],
   "web_accessible_resources": [
-    "inpage.js"
+    "inpage.js",
+    "*"
```

Comment:
I don't think we want to make *all* resources web-accessible. This seems like a potential privacy breech.

I think you should specifically list the files needed to display the page statuses (html, images).